modules/@apostrophecms/area/ui/apos/components/AposAreaContextualMenu.vue

```diff
@@ -10,6 +10,8 @@
       :button="buttonOptions"
       :popover-modifiers="inContext ? ['z-index-in-context'] : []"
       :menu-id="menuId"
+      @open="buttonOpen = true"
```

Comment:
This menu can get toggled open in two ways: via CSS classes injected by the parent, or via its own button (used when it's in an empty area). In both cases I need to trigger a fresh build of the menu so it includes the latest clipboard.